.github/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 4. AIRAC (2313) - Removed Heathrow (EGLL) MAY SIDs - thanks to @TheoBearman
 5. AIRAC (2313) - Removed Manchester (EGCC) MONTY SIDs and moved the D079P definition from EGCC to EGGP - thanks to @TheoBearman
 6. AIRAC (2313) - Reinstated Upavon (EGDJ) - thanks to @danielbutton (Daniel Button)
+7. AIRAC (2313) - Added Cranfield (EGTC) new apron - thanks to @AliceFord (Alice Ford)
```

Comment:
```suggestion
7. AIRAC (2313) - Updated Cranfield (EGTC) SMR - thanks to @AliceFord (Alice Ford)
```
Sorry slightly pedantic, but I feel this more encapsulates the change since there's also a change to taxiway D, not simply adding an apron.